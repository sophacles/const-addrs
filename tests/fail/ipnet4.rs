use const_addrs::ipnet4;

fn main() {
    let _ = ipnet4!("192.168.1.1/300");
}
