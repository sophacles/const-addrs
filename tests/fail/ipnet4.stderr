error: invalid IP address syntax
 --> tests/fail/ipnet4.rs:4:21
  |
4 |     let _ = ipnet4!("192.168.1.1/300");
  |                     ^^^^^^^^^^^^^^^^^
